extensions/adobe/experience/customerJourneyManagement/messageexecution.schema.json

```diff
@@ -84,6 +84,16 @@
           "title": "AJO Campaign Version ID",
           "type": "string",
           "description": " Version ID of the AJO Campaign that triggered this message execution."
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageExecution/ajoCampaignActionID": {
+          "title": "AJO Campaign Action ID",
+          "type": "string",
+          "description": "Action ID of the AJO Campaign that triggered this message execution."
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageExecution/batchInstanceID": {
+          "title": "Batch Instance ID for scheduled Journey or AJO Campaign Action",
```

Comment:
@chavada-adobe Can this title be shortened and put description in description part?